pypeit/core/flexure.py

```diff
@@ -169,9 +177,29 @@ def spec_flex_shift(obj_skyspec, sky_file, spec_fwhm_pix=None, mxshft=20, excess
 
     msgs.warn("If we use Paranal, cut down on wavelength early on")
```

Comment:
Sorry - just noticed this... Do you think this should be a comment, rather than a user-facing method?

Reply:
Or perhaps a `# TODO`?

Reply:
Not really sure what this message is about. It has been here for a long time. I can remove it.

Reply:
That would be fine by me - as you wish!